File: Editors/DisabledEditorsColor/disablededitorscs-zip/DisabledEditorsCS/DisabledEditors/MyTextBox.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Telerik.WinControls.UI;
using System.Windows.Forms;
using System.Drawing;

namespace DisabledEditors
{
    public class MyTextBox : HostedTextBoxBase
    {
        public MyTextBox()
        {
            this.SetStyle(ControlStyles.UserPaint, true);
        }

        protected override void OnPaint(PaintEventArgs e)
        {
            SolidBrush drawBrush = new SolidBrush(Color.Red);
            e.Graphics.DrawString(Text, Font, drawBrush, -1f, -1f);
        }
    }
}

File: Editors/DisabledEditorsColor/disablededitorsvb-zip/DisabledEditorsVB/DisabledEditors/MyTextBox.vb
Imports System.Text
Imports Telerik.WinControls.UI

Namespace DisabledEditors
	Public Class MyTextBox
		Inherits HostedTextBoxBase
		Public Sub New()
			Me.SetStyle(ControlStyles.UserPaint, True)
		End Sub

		Protected Overrides Sub OnPaint(ByVal e As PaintEventArgs)
			Dim drawBrush As New SolidBrush(Color.Red)
			e.Graphics.DrawString(Text, Font, drawBrush, -1f, -1f)
		End Sub
	End Class
End Namespace
